### ISSUE TITLE
False positive E1124 (redundant-keyword-arg) on standard library

### ISSUE THREAD

**Issue by pylint-bot** [PyCQA/pylint#231]
Originally reported by: **Julian O (BitBucket: [oddthinking](http://bitbucket.org/oddthinking), GitHub: @oddthinking?)**

---

I am getting a redundant-keyword-arg error on a line of code that looks innocent to me.

According the manual, the signature of collections.deque is:

```
#!python

class collections.deque([iterable[, maxlen]])
```

which is an odd syntax, but the meaning is clear.

Here is some code, in a file called experiments\pylintdequetest.py, that passes the iterable by position and the maxlen as a keyword argument.

```
#!python

""" Test case for pylint. """

from collections import deque
deque([], maxlen=15)
```

When I run pylint, this gives what I believe to be a spurious error:

```
#!
>pylint pylintdequetest.py
No config file found, using default configuration
************* Module experiments.pylintdequetest
E:  4, 0: Argument 'maxlen' passed by position and keyword in constructor call (redundant-keyword-arg)
[...]
```

Here are my version details, but note that I upgraded astroid since the last pylint release, in response to a previous issue I raised.

```
#!
>pylint --version
pylint-script.py 1.2.0,
astroid 1.1.0, common 0.61.0
Python 2.7.5 (default, May 15 2013, 22:43:36) [MSC v.1500 32 bit (Intel)]
```

---
- Bitbucket: https://bitbucket.org/logilab/pylint/issue/231
